### PR TITLE
use MD5 as environment variable

### PR DESCRIPTION
**Pull request by gautric**
some md5 command line is different into some OS

### CODE REVIEW

File: dist/tools/toolchains/build_gnuarm.sh [hunk 1]
```diff
@@ -25,6 +25,9 @@ MAKE_THREADS=-j4
 DOWNLOADER=wget
 DOWNLOADER_OPTS="-nv -c"
 
+MD5=md5sum
```
Comment (aabadie) [inline dist/tools/toolchains/build_gnuarm.sh]: I guess you are targeting OSX which doesn't provide md5sum ?
Then in this case, why not using a conditional case for Darwin and another for Linux using the output of `uname` command ?
See `boards/arduino-due/dist/flash.sh` for an example.

Comment (gautric) [inline dist/tools/toolchains/build_gnuarm.sh]: it is my next commit

**Comment by aabadie** [comment]
Changes look to me now. I don't have MacOS for testing though.

Can you squash your commits ?

Another thing (for future contributions), you should avoid to PR your fork master branch to the upstream master one directly. Just create a dedicated branch on your fork.

**Comment by gautric** [comment]
I got it,
squashing done

**Comment by aabadie** [comment]
Thanks @gautric, to be fully compliant with RIOT committing rules, I amended the commit message and force pushed to your branch.
Hope this is fine.

ACK

**Comment by aabadie** [comment]
Murdock is green, go!

**Comment by aabadie** [comment]
Thanks @gautric !

**Pull request merged**